tests/python/22_verify/main.py

```diff
@@ -0,0 +1,78 @@
+import sys
+import uuid
+from xrt_binding import * # found in PYTHONPATH
+from ert_binding import * # found in PYTHONPATH
+sys.path.append('../') # utils_binding.py
+from utils_binding import *
+
+def runKernel(opt):
+
+    khandle = xrtPLKernelOpen(opt.handle, opt.xuuid, "hello:hello_1")
+
+    boHandle1 = xclAllocBO(opt.handle, opt.DATA_SIZE, 0, opt.first_mem)
+    bo1 = xclMapBO(opt.handle, boHandle1, True)
+    ctypes.memset(bo1, 0, opt.DATA_SIZE)
+
+    boHandle2 = xclAllocBO(opt.handle, opt.DATA_SIZE, 0, opt.first_mem)
+    bo2 = xclMapBO(opt.handle, boHandle2, True)
+    ctypes.memset(bo2, 0, opt.DATA_SIZE)
+
+    xclSyncBO(opt.handle, boHandle1, xclBOSyncDirection.XCL_BO_SYNC_BO_TO_DEVICE, opt.DATA_SIZE, 0)
+    xclSyncBO(opt.handle, boHandle2, xclBOSyncDirection.XCL_BO_SYNC_BO_TO_DEVICE, opt.DATA_SIZE, 0)
+
+    print("Original string = [%s]\n" % bo1.contents[:].decode("utf-8"))
+    print("Original string = [%s]\n" % bo2.contents[:].decode("utf-8"))
+
+    print("Kernel start command issued through xrtKernelRun")
+    rhandle1 = xrtKernelRun(khandle, boHandle1)
+    rhandle2 = xrtKernelRun(khandle, boHandle2)
+
+    print("Now wait until the kernel finish")
+    xrtRunWait(rhandle1)
+    xrtRunWait(rhandle2)
```

Comment:
Close the run handles when done, yeah, xrtKernelClose should do this, but it doesn't.

Reply:
Fixed